lib/koans/10_map_sets.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MapSets do
+  use Koans
+
+  @intro "My name is Set, MapSet."
+
+  @set MapSet.new([1, 2, 3, 4, 5])
+
+  koan "I am merely another collection but, you can perform some operations on me" do
+    new_set = MapSet.new(@set, fn x -> 3 * x end)
```

Comment:
I like the description of this, but the first example still feels a little loaded to me. It's illustrating

1. how to initialize a set
1. that you can provide a function to change the set
1. that you can check a set for a member

I tend to think of the learning progression as
1. a set is like a list - show a set initialized with list and how it's like a list
1. however a set does not allow duplicates - show a set initialized with dups and show that they aren't in the set
1. further lessons about sets such as init with operation

That said, I don't feel so strongly that I'd recommend against merging 😄 . We can always improve later!